python/paddle/fluid/io.py

```diff
@@ -292,6 +292,19 @@ def save_persistables(executor, dirname, main_program=None, filename=None):
         filename=filename)
 
 
+def _delete_initialize_op(var):
```

Comment:
I think this should be done in ParallelExecutor.py. Since you pass in a Program class, Parallel_executor.py can compile the Program in-place